ParisBNF/et/BNFet50.xml

```diff
@@ -42,85 +42,85 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                      <note>The chapter division is the same as in the other
                   manuscripts of <ref type="work" corresp="LIT1340EnochE"/>, but it is only marked at the beginning.</note>
                      <note>The text of this manuscript agrees always with one of the five manuscripts
-                  used for <bibl> 
+                  used for <bibl>
                            <ptr target="bm:Dillmann1851Henoch"/>
                         </bibl>.</note>
                   </msItem>
-            
+
                   <msItem xml:id="ms_i2">
                      <locus from="20r" facs="f41"/>
                      <title ref="LIT1719Bookso" type="complete"/>
                      <textLang mainLang="gez"/>
                      <note>There are no chapter divisions except the marking of European chapters in the margins.</note>
-              
+
                      <msItem xml:id="ms_i2.1">
                         <locus from="20r" facs="f41"/>
                         <title ref="LIT2697Sam" type="complete" xml:lang="gez">መጽሐፈ፡ ነገሥት፡ ቀዳማዊ።</title>
                         <textLang mainLang="gez"/>
-                        <colophon xml:lang="gez" xml:id="coloph1">ለመጽሐፈ፡ ነገሥት፡ ቀዳማዊ፡ ወጠኖ፡ 
+                        <colophon xml:lang="gez" xml:id="coloph1">ለመጽሐፈ፡ ነገሥት፡ ቀዳማዊ፡ ወጠኖ፡
                   <persName ref="PRS11010Samuel" role="scribe">ሳሙኤል</persName>፡
                   ወፈጸምዎ፡ <persName ref="PRS11011Natan" role="scribe">ናታን</persName>፡ ወ<persName ref="PRS11012Gad" role="scribe">ጋድ</persName>።</colophon>
-                        <note>See for the variants of the first chapter of <ref target="#ms_i2.1"/> 
-                           <ref target="#"/>.</note> 
+                        <note>See for the variants of the first chapter of <ref target="#ms_i2.1"/>
+                           <ref target="#"/>.</note>
```

Comment:
why empty ref target? needed? can be added?

Reply:
Concerning this and a few more corrections below (I could have told this in the beginning), I revised only the segment with the genealogy and around. I did not do anything else on this item.

Reply:
@thea-m is it ok to approve and merge or do you think the empty targets can be removed or filled? thank you!